docs/endpoints.md

```diff
@@ -82,6 +82,100 @@ Each signature response contains the following fields:
     format. Each signer uses a different format, so refer to their
     documentation for more information.
 
+## /sign/files
+
+### Request
+
+Request to sign multiple files. The files to sign are passed in
+the request body using the JSON format described below.
+
+The request body is an array of signature requests, to allow for
+batching signatures into a single API request. The parameters are:
+
+-   **keyid**: allows the caller to specify a key to sign the data with.
+    This parameter is optional, and Autograph will pick a key based on
+    the caller\'s permission if omitted.
+-   **options**: a JSON object used to pass signer-specific options in
+    the request. Refer to the documentation of each signer to find out
+    which options they accept.
+-   **files**: an array of dicts of file name and base64 encoded content to sign. For example:
+
+```json
+[
+    {
+        "content": "UEsDBBQACAAIAAAAAAAAAAAAAAAAAAAAAAATAAAAQW5kcm9pZE1hbmlmZXN0LnhtbKSYS2ybx7XHf0PqbVmW4...BwAACigAAAAA",
+        "name": "sphinx_1.7.2-1.dsc"
+    },
+    {
+        "content": "UEsDBBQACAAIAAAAAAAAAAAAAAAAAAAAAAATAAAAQW5kcm9pZE1hbmlmZXN0LnhtbKSYS2ybx7XHf0PqbVmW4...BwAACigAAAAA",
+        "name": "sphinx_1.7.2-1.changes"
+    }
+]
+```
+
+The number of files must be between 1 and 32 inclusive. All characters
+in a file name must be an alphanumeric character, dash, underscore or
+dot. The file name must start with an alphanumeric character and dots
+cannot occur next to each other in the file name.
+
+example:
+
+``` bash
+POST /sign/file
```

Comment:
Should this be `/sign/files`?

Reply:
d'oh yeah, nice catch! Fixed in 449aa5eb